client/jetpack-cloud/sections/partner-portal/issue-license-v2/licenses-form/index.tsx

```diff
@@ -174,6 +175,16 @@ export default function LicensesForm( {
 		);
 	};
 
+	const isSearchResultEmpty = useMemo( () => {
+		return (
+			productSearchQuery !== '' &&
+			plans.length === 0 &&
+			products.length === 0 &&
+			wooExtensions.length === 0 &&
+			backupAddons.length === 0
+		);
+	}, [ productSearchQuery, plans, products, wooExtensions, backupAddons ] );
```

Comment:
[Non-blocker] We can also use filteredProductsAndBundles to check if the result is empty instead of doing it individually.

```suggestion
    const isSearchResultEmpty = useMemo( () => {
		return productSearchQuery !== '' && filteredProductsAndBundles.length === 0;
	}, [ productSearchQuery, filteredProductsAndBundles.length ] );
```

Reply:
Oh, great eye! I made that change and decided to remove the `isSearchResulEmpty` function call as the code is way more simpler now.